Tasks/GradleV2/task.loc.json

```diff
@@ -16,8 +16,8 @@
   "author": "Microsoft Corporation",
   "version": {
     "Major": 2,
-    "Minor": 163,
-    "Patch": 2
+    "Minor": 173,
```

Comment:
Could you please revert Gradle task version bump since there's no changes at the moment?

Reply:
There is one : the visibleRule condition has changed.